apps/partners/src/components/ProfilePopover.tsx

```
@@ -0,0 +1,100 @@
+              )}
+            </button>
+          ))}
+        </div>
+        <div className="border-t border-gray-200 p-2">
+          <button
+            className="w-full px-4 py-2 rounded-md bg-blue-500 hover:bg-blue-600 text-white font-semibold"
+            onClick={() => {
+              setOpen(false)
+              router.push("/organization")
+            }}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)